app/views/articles/_sidebar.html.erb

```diff
@@ -26,6 +26,14 @@
                 </div>
               </div>
             <% end %>
+            <div class="sponsors-love-message">
+              We are grateful for wonderful sponsors who help sustain the dev community.
```

Comment:
s/dev/DEV ?

Reply:
Interesting. Here I'm re-adding the same text I accidentally removed before. So lowercase dev was always in place. But made your suggested fix.